PORTING-ADDONS-TO-V2.md

```diff
@@ -249,7 +253,7 @@ Now that we've separated the test-app and docs app concerns from the addon, we c
     ],
     "scripts": {
       "build": "rollup --config",
-      "prepublishOnly": "rollup --config",
+      "prepublishOnly": "cp ../README.md ../LICENSE.md . && rollup --config",
```

Comment:
Let's suggest `npm-run-all` instead (most apps/addons already have it, it's in the default ember-cli blueprint)

Reply:
We could also integrate copying into the rollup build itself, e.g. with something like https://github.com/vladshcherbin/rollup-plugin-copy. To not split concerns between npm script and rollup.

At least I was thinking about this for the blueprint (https://github.com/embroider-build/addon-blueprint/issues/8)

Reply:
@simonihmig the only downside is that it will do so even during regular build step. Can't say it's big deal/an issue, just a side effect of such approach

Reply:
had another idea:

use `before:init` hook for the `release-it` like so:

```json
"release-it": {
  "hooks": {
    "before:init": "cp README.md LICENSE.md addon/"
  }
}
```

that way, release process concerns are specific to the release tooling.
Since we copy files form monorepo root folder to the addon folder, I think ideally addon should not care about anything outside of the addon folder itself.

made PR https://github.com/ember-animation/ember-animated/pull/466/files so you may check it out how this looks like in practice.

Reply:
In general, it seems quite tricky to maintain all addons (at scale) release process in package.json file.
Thinking about single source of truth (may be release.js at the start), and later, could be extracted as an ember ecosystem package.

```js
{
  release: "npx ember-addon-release@2 --release"
}
```